app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx

```diff
@@ -7,14 +7,18 @@ import { HeaderFont } from '../../../../components'
 import { tailwind } from '../../../../tailwind'
 import { translate } from '../../../../translations'
 import { AboutScreen } from './screens/AboutScreen'
+import { ChangePinScreen } from './screens/ChangePinScreen'
 import { CommunityScreen } from './screens/CommunityScreen'
+import { ConfirmPinScreen } from './screens/ConfirmPinScreen'
 import { RecoveryWordsScreen } from './screens/RecoveryWordsScreen'
 import { SettingsScreen } from './SettingsScreen'
 
 export interface SettingsParamList {
   SettingsScreen: undefined
   CommunityScreen: undefined
   RecoveryWordsScreen: { words: string[] }
+  ChangePinScreen: { pinLength: number, words: string[] }
+  ConfirmPinScreen: { pin: string, words: string[] }
```

Comment:
Decided to go by creating new similar component, instead of using create wallet pages

- they look similar but logic/UI not really identical.
- they are in different navigation container